htdocs/commande/list.php

```diff
@@ -2573,7 +2573,7 @@
 					if ($reshook == 1) {
 						// for add information
 						$text_info .= $hookmanager->resPrint;
-					} elseif ($reshook == 0) {
```

Comment:
i made an error by validating a new hook into an already released branch. i revert it.
You can submit the hook into develop branch.
Also the fix is not correct. We should have
```

if ($reshook == 1) {
						// for add information
						$text_info = $hookmanager->resPrint;
					} elseif ($reshook == 0) {
						$text_info .= $hookmanager->resPrint;

```